### PR TITLE
Improve readme instructions issue#35

### PR DESCRIPTION
**Pull request by nelsonic**
+ Re-organised the Readme into clearerer headers. #35 
+ Updated screenshots. #111 
+ Included links to `new` "**learn-ngrok**" tutorial: https://github.com/dwyl/learn-ngrok
+ added **`.env_sample`** file to speed up setup.
+ clarified instructions for setting Private Key as (_multi-line_) environment variable 
  see: https://github.com/dwyl/learn-environment-variables/issues/17

### CODE REVIEW

**Comment by SimonLab** [comment]
We might need to update the Erlang version on Travis:
image

We can define the version in our travis.yml file, see https://docs.travis-ci.com/user/languages/erlang/

However I don't think the error is directly linked to the Erlang version, I'll try to see what cause the exit status to 1:
image

**Comment by nelsonic** [comment]
@Cleop / @iteles / @SimonLab 
I made some pretty _substantial_ updates to the readme
including an end-to-end set of screenshots and installation instructions.
Not going to add anything `else` now, so feel free to review and give feedback.
For reading the `README.md` changes, see: 
https://github.com/dwyl/github-backup/blob/improve-readme-instructions-issue%2335/README.md

Thanks! ❤️

File: .env_sample [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+export GITHUB_APP_ID=YOUR_GITHUB_APP_ID
+
+# GITHUB_APP_NAME what you called your app.
+export GITHUB_APP_NAME=NAME_OF_THE_GITHUB_APP
+
+# SECRET_KEY_BASE is required for Auth Cookie:
+export SECRET_KEY_BASE=run:mix_phx.gen.secret-in-terminal-and-paste-result-here
+
+# AWS:
+export AWS_ACCESS_KEY_ID=aws-access-key
```
Comment (Cleop) [inline .env_sample]: Tiny observation but perhaps `aws-access-key` could say `aws-access-key-id` in line with the key name?

File: README.md [hunk 2]
```diff
@@ -1,170 +1,508 @@
...
+as the `.env` file is "ignored" in the_
+[`.gitignore` file](https://github.com/dwyl/github-backup/blob/778d1e311a37564721989e842412880994eb2b5d/.gitignore#L28)
+
+
+
+### Create a GitHub Application
+
+The role of the Github application is to
+send **notifications** when **events** occur in your repositories. <br />
+For example you can get be notified
```
Comment (Cleop) [inline README.md]: Typo, either needs to be 'For example you can ~get~ be notified' or 'For example you can get ~be~ notified'.

File: README.md [hunk 3]
```diff
@@ -1,170 +1,508 @@
...
-The `github-backup` server will receive events from Github and then save the details or edits to an external database.
+#### "Source" the `.env` File
 
-- Clone the repository _to your personal computer_:
-  ```
-  git clone git@github.com:dwyl/github-backup.git && cd github-backup
-  ```
-- Define the local environment variables:
+Now that you have defined all the necessary
+environment variables in your `.env` file, <br /
```
Comment (Cleop) [inline README.md]: This `<br /` is not closed, either close it or remove it 🤔

File: .env_sample (hunk 1 above)
Comment (nelsonic) [reply .env_sample]: thanks for your attention to detail @Cleop 🎉

File: README.md (hunk 3 above)
Comment (nelsonic) [reply README.md]: fixed.

File: .gitignore [hunk 4]
```diff
@@ -27,3 +27,4 @@ npm-debug.log
 /config/*.secret.exs
 .env
 cover
+*.pem
```
Comment (SimonLab) [inline .gitignore]: I'm not sure if this line is still necessary :thinking: 
I think it's useful if user generate a private key at the root of the application :+1:

**Pull request merged**